### PR TITLE
[Jetsurvey] Issue460/superhero questions

### PR DESCRIPTION
**Pull request by angeles-bilbao6**
Added images for Superhero questions
Updated space between question and button for Date Question

https://github.com/android/compose-samples/issues/460

Super hero / movie questions

 Images missing for super hero and movie questions
 Space between "Select date" and date button should be 24.dp

Before
image
image

After
image
image

### CODE REVIEW

**Comment by florina-muntenescu** [comment]
The images used in the project should come from the Dev Challenge comicstrip. We're not allowed to accept these superhero images due to copyright issues.

**Comment by c5inco** [comment]
Yep, I'm waiting on approval from the agency to reuse imagery from the Dev Challenge comicstrip since the assets will be open-source.

**Comment by angeles-bilbao6** [comment]
Replaced superheroes with comic characters

image

**Review by florina-muntenescu — APPROVED** [review]

**Pull request merged**
